lib/src/utils/utils.dart

```diff
@@ -93,3 +94,24 @@ Map<String, T>? removeAliasPrefix<T>(Map<String, T?>? map) {
   });
   return mapResult;
 }
+
+/// Generate a random seed
+String generateRandomSeed() {
+  var seed = '';
+  const chars = 'abcdef0123456789';
+  final rng = Random.secure();
+  for (var i = 0; i < 64; i++) {
+    // ignore: use_string_buffers
+    seed += chars[rng.nextInt(chars.length)];
+  }
+  return seed;
+}
+
+/// Generate a random AESKey
+String generateRandomAESKey({int length = 32}) {
```

Comment:
Where are you going to use this method ? From the wallet ?

Reply:
`MessagingService` class